.github/workflows/facilities_build.yml

```diff
@@ -46,6 +43,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Load secrets
```

Comment:
@damonmcc @fvankrieken @sf-dcp How we feel about copy-pasting this block all over? I've never extracted a job step into anything reusable, so I thought I'd ask the experts if it's possible/worthwhile.

Reply:
yea I don't think steps can be be extracted to be reusable like jobs can be (e.g. all our product build actions are jobs and can be called in other workflows)

sure helps using the `1password` action already! not really any unique logic here so copy-pasting these action inputs doesn't seem too bad, plus the envars needed for an action may vary?

Reply:
Yeah I'd say from my brief searching in the past, this is about as reusable as it gets

Reply:
Cool - off to the races then (ie copy-pasting this everywhere)

Reply:
[whatever the opposite of ✂️ is]

Reply:
🍝

Reply:
🪨 + 📄